src/qibosoq/log.py

```diff
@@ -3,13 +3,20 @@
 import logging
 import logging.handlers
 import os
+import sys
 from typing import Tuple
 
 import qibosoq.configuration as cfg
 
 
 def configure_logger(name: str, filename: str, backup_count: int):
     """Create and configure logger."""
+    # if the log directory does not exsist, create it
+    if sys.platform == "linux":
+        dir_path = "/".join(filename.split("/")[:-1])
+        if not os.path.exists(dir_path):
+            os.mkdir(dir_path)
```

Comment:
You can convert all of this to `pathlib`:

```suggestion
        dir_path = Path(filename).parent
        if dir_path.exists():
            dir_path.mkdir()
```

Reply:
This is why I was restring things to linux, because of the '/', but with pathlib is nicer.
Anyway, this will always run only on the server that is linux

Reply:
fixed